package.json

```diff
@@ -4,7 +4,7 @@
   "description": "API server for the Screwdriver.cd service",
   "main": "index.js",
   "scripts": {
-    "pretest": "eslint .",
+    "pretest": "eslint --max-warnings 440 .",
```

Comment:
what is this for?

Reply:
we have lots of warnings, 437 all of them are number of function arguments and number of lines in a function. We should strive to refactor current functions and make sure that this number does not increase. Any new code should not fall into this state.